components/build-service/base/rbac/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
 resources:
 - build-maintainer.yaml
+- build-team-can-view-sa-secret.yaml
```

Comment:
I think we cannot view secrets.

Reply:
yeah but that doesn't help with debugging tbh

Reply:
only verb list is granted for Secret resource